app/src/main/java/de/westnordost/streetcomplete/data/user/achievements/AchievementsModule.kt

```diff
@@ -702,6 +707,7 @@ object AchievementsModule {
                     "AddCyclewayPartSurface",
                     "AddFootwayPartSurface",
                     "AddBenchBackrest",
+                    "AddBarrierType",
```

Comment:
IMO that's a bit much. I think "for cars" and "outdoors" could be removed.

Reply:
I think that for both it actually matters and matches - for example for cars betwen `barrier=height_restrictor` vs `barrier=gate`. And even between different `barrier=stile` variants there is a difference.

But I can remove if you want.

Reply:
No strong feeling about this. Achievements are 🤷  anyway